README.md

```diff
@@ -45,17 +61,19 @@ Span #19
 Also, the counters displayed on http://localhost:55679/debug/tracez should increase as
 you generate more traces with the telemetrygen command.
 
+The dashboard on https://ui.honeycomb.io should display data for the `telemetrygen` dataset.
+
 ## Start Rails application
 
 ```bash
 export OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318"
-export OTEL_EXPORTER_OTLP_PROTOCOL="http/json"
-export OTEL_EXPORTER_OTLP_TRACES_PROTOCOL="http/json"
```

Comment:
I was under the impression that both OTEL_EXPORTER_OTLP_PROTOCOL and OTEL_EXPORTER_OTLP_TRACES_PROTOCOL had to be set to http/json, (but I could be wrong as I wasn't there when these vars were defined in the readme)

Reply:
Correct, when we tested we set both.  But according to Howard defining `OTEL_EXPORTER_OTLP_PROTOCOL` will set both `OTEL_EXPORTER_OTLP_TRACES_PROTOCOL` and `OTEL_EXPORTER_OTLP_METRICS_PROTOCOL`.

I tested myself and it works as Howard said.